packages/client/libclient-py/quickmpc/pandas/share_data_frame.py

```diff
@@ -151,27 +178,89 @@ def join_list(self, others: List["ShareDataFrame"], *, debug_mode=False)\
                               True, ShareDataFrameStatus.EXECUTE)
 
     def sum(self, columns: list) -> "ShareDataFrame":
+        """列の総和を取得する
+
+        Parameters
+        ----------
+        columns: list
+            計算に用いる列番号(1-index)
+
+        Returns
+        ----------
+        Result
+            結果のDataFrame
+        """
         res = self.__qmpc_request.sum([self.__id], columns)
         return ShareDataFrame(res.job_uuid, self.__qmpc_request,
                               True, ShareDataFrameStatus.EXECUTE)
 
     def mean(self, columns: list) -> "ShareDataFrame":
+        """列の平均を取得する
+
+        Parameters
+        ----------
+        columns: list
+            計算に用いる列番号(1-index)
+
+        Returns
+        ----------
+        Result
+            結果のDataFrame
+        """
         res = self.__qmpc_request.mean([self.__id], columns)
         return ShareDataFrame(res.job_uuid, self.__qmpc_request,
                               True, ShareDataFrameStatus.EXECUTE)
 
     def variance(self, columns: list) -> "ShareDataFrame":
+        """列の分散を取得する
+
+        Parameters
+        ----------
+        columns: list
+            計算に用いる列番号(1-index)
+
+        Returns
+        ----------
+        Result
+            結果のDataFrame
+        """
         res = self.__qmpc_request.variance([self.__id], columns)
         return ShareDataFrame(res.job_uuid, self.__qmpc_request,
                               True, ShareDataFrameStatus.EXECUTE)
 
     def correl(self, columns1: list, columns2: list) -> "ShareDataFrame":
+        """列同士の分散を取得する
+
+        Parameters
+        ----------
+        columns1: list
+            計算に用いる左項の列番号(1-index)
+        columns2: list
+            計算に用いる右項の列番号(1-index)
+
+        Returns
+        ----------
+        Result
+            結果のDataFrame
```

Comment:
A little more detail would be nice.
The following examples are not appropriate, but are intended to share an image.
```
Result
            ShareDataFrame
            Result[i][j] : columns1[i] と columns2[j] の相関係数
```

Reply:
Thank you very much!
Very nice, but one of the `column1` in correl is probably a mistake for `column2`.
Everything else is fine.

Reply:
wow! I fixed it. Thanks! 6c231fd
I don't know why the commit is not shown on this page, but I will investigate later.